jf_agent/git/bitbucket_cloud_adapter.py

```diff
@@ -236,7 +226,7 @@ def get_pull_requests(
                                 or 'repository' not in api_pr['destination']
                                 or not api_pr['destination']['repository']
                             ):
-                                agent_logging.log_and_print_error_or_warning(
+                                agent_logging.log_standard_error(
                                     logger, logging.WARN, msg_args=[api_pr['id']], error_code=3030
```

Comment:
Nit: Should be "WARNING"

Reply:
I see this is used in several places. U2u if you want to change across the project, this is NB.

Reply:
Yeah I wasn't really sure what the difference was. I'll replace them all with `WARNING`, since it seems like the proper styling